sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -0,0 +1,331 @@
+
+Admininistrator User Manual
+============================
+**Who is an "administrator"?**
+
+A person who :
+    * helps to plan and oversee a project using KA Lite
+    * installs and updates KA Lite
+    * can create coach logins, Learner logins, download videos and language packs
+
+Administrator Glossary
+-------------------------------------------
+For users that choose to register online, there are some important terms to familiarize yourself with so that you understand how the flow of data works between installations and the online data hub.
+
+**Sharing Network**
```

Comment:
@jamalex it would be good to start thinking about how we wanna migrate the documentation once we implement the new flexmodels.